api/src/main/java/org/pmiops/workbench/exfiltration/EgressRemediationService.java

```diff
@@ -253,4 +278,99 @@ private void stopUserRuntimes(String userEmail) {
+            // Ideally we would use Resolution = Unresolved here, but due to a misconfiguration of
+            // RW Jira, transitioning to Won't Fix / Duplicate do not currently resolve an issue.
+            String.format(
+                "\"%s\" ~ \"%s\""
+                    + " AND \"%s\" ~ \"%s\""
+                    + " AND status not in (Done, \"Won't Fix\", Duplicate)"
+                    + " ORDER BY created DESC",
+                IssueProperty.EGRESS_VM_PREFIX.key(),
+                event.getUser().getRuntimeName(),
+                IssueProperty.RW_ENVIRONMENT.key(),
+                envShortName));
+
+    if (results.getIssues().isEmpty()) {
+      jiraService.createIssue(
+          IssueType.TASK,
+          jiraEventDescription(event, action),
+          ImmutableMap.<IssueProperty, Object>builder()
+              .put(
+                  IssueProperty.SUMMARY,
+                  String.format(
+                      "(%s) Investigate egress from %s",
+                      jiraDateFormatter.format(clock.instant()), event.getUser().getUsername()))
+              .put(IssueProperty.EGRESS_VM_PREFIX, event.getUser().getRuntimeName())
+              .put(IssueProperty.RW_ENVIRONMENT, envShortName)
+              .put(IssueProperty.LABELS, new String[] {"high-egress"})
+              .build());
+    } else {
+      IssueBean existingIssue = results.getIssues().get(0);
+      if (results.getIssues().size() > 1) {
+        log.warning(
+            String.format(
+                "found multiple (%d) open Jira tickets for the same user VM prefix, updating the most recent ticket",
+                results.getIssues().size()));
+      }
+      jiraService.commentIssue(existingIssue.getId(), jiraEventComment(event, action));
+    }
+  }
+
+  private String jiraEventDescription(DbEgressEvent event, EgressRemediationAction action) {
+    Optional<DbUser> user = Optional.ofNullable(event.getUser());
+    SumologicEgressEvent originalEvent =
+        new Gson().fromJson(event.getSumologicEvent(), SumologicEgressEvent.class);
+    return String.format(
+        String.format("Notebook server VM prefix: %s\n", originalEvent.getVmPrefix())
+            + String.format(
+                "User running notebook: %s\n\n", user.map(DbUser::getUsername).orElse("unknown"))
+            + jiraEventDescriptionShort(event, action));
+  }
+
+  private String jiraEventComment(DbEgressEvent event, EgressRemediationAction action) {
+    return "Additional egress detected\n\n" + jiraEventDescriptionShort(event, action);
+  }
+
+  private String jiraEventDescriptionShort(DbEgressEvent event, EgressRemediationAction action) {
+    Optional<DbWorkspace> workspace = Optional.ofNullable(event.getWorkspace());
+    SumologicEgressEvent originalEvent =
+        new Gson().fromJson(event.getSumologicEvent(), SumologicEgressEvent.class);
+    return String.format(
+        String.format("Action taken: %s\n\n", action)
+            + String.format(
```

Comment:
Would it be also helpful to put user email here? 
Or another good thing good to have is add user id into admin/user table. Then I can find that user by runtime name

Reply:
These details are included in the longer description:

`jiraEventDescription()` -> used for the initial ticket (depends on `jiraEventDescriptionShort`)
`jiraEventComment()` -> used to append to an existing ticket (depends on `jiraEventDescriptionShort`)

For the latter, including username etc is redundant, since this is the criteria for matching the ticket. Those details will already be in the ticket description.

Reply:
RE user ID - VM prefix will already be in the ticket description and title. Is this sufficient for that need?

Reply:
1: SG. 
2: User Id, that is completely separate question. Sometimes I want to look up user by runtime name. Now I have to connect to db and query by VM name. It would be nice to make user_id in admin user table in the UI and searchable.

Reply:
VM name is just derived from the user ID: "all-of-us-{userId}", it is not stored in the DB. If you're using the admin UI, you should just use the username, which is included in the ticket and is already searchable